### DISCUSSION TITLE
[Enhancement] [dropdown] Dropdown！ Whether to support popup in the lower right corner of the mouse inside an element ？

### DISCUSSION THREAD

**Discussion by balck-paint** [element-plus/element-plus#8766] [Q&A] [answered]
Feature type: **`Enhancement`**

## Details

- Enhanced component name: `dropdown`

## Description

这个右键菜单是否支持在鼠标点击位置的右下角弹出，我看这个默认是在元素外部弹出的。看了文档是有用popper.js这个插件，又看了这个插件的文档，里面有个inner应该可以配置，但是我设置了之后没有效果。所以这个如果想实现这个功能可以通过配置实现吗？

如 <a href="https://codesandbox.io/s/q4lnii?file=/src/demo.vue">antd vue下拉菜单中的右键菜单</a>

### Additional comments

_(empty)_

**Comment by rzzf** [comment, marked as answer]
现在它支持了：https://github.com/element-plus/element-plus/pull/22056
